components/todo/TodoContainer.vue

```diff
@@ -14,9 +14,14 @@
       />
 
       <!-- List of todos -->
-      <div v-for="todo in todos" :key="todo.id" class="card-item">
-        <TodoItem :todo="todo" />
-      </div>
+      <template v-if="todos.length">
```

Comment:
We can use a computed maybe named `isTodoAvailable`. Maybe nitpicking but gives us a good overview of the conditional.